data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/consumer/CloudEventOverridesMutator.java

```diff
@@ -46,7 +46,7 @@ private void applyCloudEventOverrides(CloudEventBuilder builder) {
     }
 
     private void applyKafkaMetadata(CloudEventBuilder builder, Number partition, Number offset) {
-        builder.withExtension("knativekafkapartition", partition);
-        builder.withExtension("knativekafkaoffset", offset);
+        builder.withExtension("knkafkapartition", partition);
```

Comment:
@Cali0707 can you check w/ the Java SDK why it is not failing on 20+ chars?
(and maybe double check w/ golang)?